spec/javascripts/components/short_form_language_switcher_spec.coffee

```diff
@@ -0,0 +1,27 @@
+do ->
```

Comment:
@chrisdolendo seems like you got this sorted out? looks like the spec is running